src/test/java/com/jcabi/beanstalk/maven/plugin/WarFileTest.java

```diff
@@ -100,4 +103,32 @@ public void checkEbextensionsValidityThrowsExceptionNoConfigFiles()
             );
         }
     }
+
+    /**
+     * Verifies that checkEbextensionsValidity runs fine
+     * when .ebextensions entry has a valid json text.
+     * @throws Exception Thrown in case of error.
+     */
+    @Test
+    public void checkValidateJson() throws Exception {
+        final ZipFile zip = Mockito.mock(ZipFile.class);
+        final ZipEntry entry = Mockito.mock(ZipEntry.class);
+        Mockito.when(zip.getEntry(".ebextensions")).thenReturn(entry);
+        @SuppressWarnings("unchecked")
+        final Enumeration<ZipEntry> enumeration =
+            Mockito.mock(Enumeration.class);
+        Mockito.doReturn(enumeration).when(zip).entries();
+        Mockito.when(enumeration.hasMoreElements()).thenReturn(true)
+            .thenReturn(false);
+        Mockito.when(enumeration.nextElement()).thenReturn(entry)
+            .thenReturn(null);
+        Mockito.when(entry.getName()).thenReturn(".ebextensions/");
+        Mockito.when(zip.getInputStream(entry)).thenReturn(
+            new ByteArrayInputStream(
+                "{\"blah\":\"blah\"}".getBytes(StandardCharsets.UTF_8)
+            )
+        );
+        final WarFile war = new WarFile(zip);
+        war.checkEbextensionsValidity();
```

Comment:
this test doesn't assert or verify anything, either add that or add information in the test description that the only verification is lack of exception